test/jdk/javax/security/auth/PrivateCredentialPermission/Serial2.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8291974
+ * @summary     PrivateCredentialPermission should not use local variable to enable debugging
+ *              implementation-dependent class
+ */
+
+import javax.security.auth.*;
+import java.io.*;
+import java.util.*;
+
+public class Serial2 implements java.io.Serializable {
```

Comment:
I don't see why this needs to implement `Serializable` (I see you probably used the other `Serial.java` test in this directory as a template. I don't think that needs it either, but you can leave that as is for now).

Reply:
I read your sentence as "leave both for now". Sorry. You mean fix in this PR and leave the other alone.